GitCommands/Logging/CommandLog.cs

```diff
@@ -85,18 +84,16 @@ public string ColumnLine
                     fileName = "git";
                 }
 
-                var pid = ProcessId == null ? "     " : $"{ProcessId,5}";
-                var exit = ExitCode == null ? "  " : $"{ExitCode,2}";
+                var pid = ProcessId == null ? "" : $"{ProcessId}";
+                var exit = ExitCode == null ? "" : $"{ExitCode}";
 
-                return $"{StartedAt:HH:mm:ss.fff} {duration,7} {pid} {(IsOnMainThread ? "UI" : "  ")} {exit} {fileName} {Arguments}";
+                return $"{StartedAt:HH:mm:ss.fff} {duration,7} {pid,5} {(IsOnMainThread ? "UI" : "  ")} {exit,2} {fileName} {Arguments}";
             }
         }
 
         public string FullLine(string sep)
         {
-            var duration = Duration == null
-                ? "running"
-                : $"{((TimeSpan)Duration).TotalMilliseconds:0}ms";
+            var duration = Duration == null ? "" : $"{((TimeSpan)Duration).TotalMilliseconds:0}";
```

Comment:
While you at it, could you please format the duration with a separator? Make it easier to read
E.g. `2346363ms` -> `2,346,363ms`

![image](https://user-images.githubusercontent.com/4403806/48319737-c3735100-e665-11e8-9ed8-c4e3b3bc7e50.png)

Reply:
Will look like the following with space as separator (as in the international standard).

![image](https://user-images.githubusercontent.com/6248932/48320038-4324ff80-e615-11e8-8b79-738f11de7ec9.png)

This change should be separated, I just want to have feedback before bedtime (and a busy week at work).

Reply:
This is what the thousands separator looks like with international/ISO number separator. For English standard, it will be ',', for traditional Europe it will be '.'. 
https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping

Reply:
Ah yes, I kind of implied, it would be formatted as per user's regional settings

Reply:
A space to ms is required too, as well as at least one another extra space for column width (two extra to handle tens of seconds too, but >= 10 could be OK to get unaligned).